### PR TITLE
Ergo S-1 OSE

### PR DESCRIPTION
**Pull request by wizarddata**
New board info: https://github.com/wizarddata/Ergo-S-1

## Board/Shield Check-list
 - [x] This board/shield is tested working on real hardware
 - [x] Definitions follow the general style of other shields/boards upstream ([Reference](https://zmk.dev/docs/development/new-shield))
 - [x] `.zmk.yml` metadata file added
 - [x] Proper Copyright + License headers added to applicable files (Generally, we stick to "The ZMK Contributors" for copyrights to help avoid churn when files get edited)
 - [x] General consistent formatting of DeviceTree files
 - [x] Keymaps do not use deprecated key defines (Check using the [upgrader tool](https://zmk.dev/docs/codes/keymap-upgrader))
 - [x] `&pro_micro` used in favor of `&pro_micro_d/a` if applicable
 - [x] If split, no name added for the right/peripheral half
 - [x] Kconfig.defconfig file correctly wraps *all* configuration in conditional on the shield symbol
 - [x] `.conf` file has optional extra features commented out
 - [x] Keyboard/PCB is part of a shipped group buy or is generally available in stock to purchase (OSH/personal projects without general availability should create a zmk-config repo instead)

### CODE REVIEW

File: app/boards/shields/ergos1_ose/ergos1_ose.zmk.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+file_format: "1"
```
Comment (caksoylar) [inline app/boards/shields/ergos1_ose/ergos1_ose.zmk.yml:1]: Looks like this file was copied from redox and not modified.

Comment (wizarddata) [reply app/boards/shields/ergos1_ose/ergos1_ose.zmk.yml:1]: You're right, I'll update that

**Comment by caksoylar** [comment]
Is there a reason you are not using the `pro_micro` interconnect definitions and not setting `requires: [pro_micro]` in the metadata? You don't seem to be using nano-specific pins but I might be wrong.

**Comment by wizarddata** [comment]
This keyboard didn't come from an existing design that used the pro-micro footprint, so referencing the gpio directly seemed to make more sense.

**Comment by caksoylar** [comment]
Changing them to be `pro_micro` would allow your shield to be used with other boards like `nrfmicro` as well, since [it has different internal pin assignments](https://github-wiki-see.page/m/joric/nrfmicro/wiki/Pinout).

**Comment by wizarddata** [comment]
It was brought to my attention that I need to make some changes to board and file names that will effect this pull request. Should I close this one and open a new PR after the changes are made?

**Comment by wizarddata** [comment]
The changes require that the name of the repository and all filenames change as well, I'm going to briefly repoen this PR to see if that change is possible.

**Comment by wizarddata** [comment]
Looks like those changes can't take after the branch name changed, so I'm going to open a new PR.

**Pull request closed without merging**